src/mixins/amalgamation-mixin.ts

```diff
@@ -287,15 +287,15 @@ export default class AmalgamationMixin extends Vue {
       if (!tingBusiness.authInfo && !tingBusiness.businessInfo) {
         return {
           type: AmlTypes.FOREIGN,
-          role: AmlRoles.AMALGAMATING,
+          role: AmlRoles.AMALGAMATING, // *** FUTURE: can we really assume this?
           corpNumber: item.corpNumber,
           legalName: item.legalName,
           foreignJurisdiction: item.foreignJurisdiction
         } as AmalgamatingBusinessIF
       } else {
         return {
           type: AmlTypes.LEAR,
-          role: AmlRoles.AMALGAMATING,
+          role: item.role, // amalgamating or holding
```

Comment:
Use the role from the item (which is what was restored from the draft).

Reply:
Example after fix:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/2d629158-ef43-4251-85e6-9e3f673d99ed)